src/lib/modules/server/apn/types.ts

```
@@ -0,0 +1,36 @@
+import type { Error, ErrorData, LibraryFailedItem, LibrarySentItem, RawResponse, SentDetail } from '$generated/types';
+
+// Types not yet generated by type-crafter (contain $ref/arrays/oneOf)
+// TODO: Remove these once type-crafter generates them
+
+export type NotificationPayload = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(`@typescript-eslint/consistent-type-definitions`)

---

[error] Expected "body" to come before "title".

(perfectionist/sort-object-types)